src/main/java/com/flightstats/hub/app/HubHost.java

```diff
@@ -1,20 +1,22 @@
 package com.flightstats.hub.app;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import com.flightstats.hub.config.AppProperty;
+import com.flightstats.hub.config.PropertyLoader;
+import lombok.extern.slf4j.Slf4j;
 
 import java.net.InetAddress;
 import java.net.UnknownHostException;
 
+@Slf4j
 public class HubHost {
```

Comment:
This would be cumbersome since it is used all over Hub. I would do this as a separate task. This  will also include getting rid of deprecated HubProvider class